book/special.qmd

```diff
@@ -851,188 +851,188 @@ The experiment shows that using the K-means algorithm with four centers has the
+% #| fig-alt: Six scatter plots are shown with y-axes from 4 degrees South to 3.97 degrees South and x-axes from 79.08 degrees West to 79.06 degrees West. The top row of plots show a very clear separation with no overlap between blue and orange datapoints, whereas the bottom low shows significant overlap.
+% #| label: fig-sprsmp
+%
+% library(patchwork)
+% autoplot(rsmp_sp, task, fold_id = c(1:3), size = 0.7) /
+%   autoplot(rsmp_nsp, task, fold_id = c(1:3), size = 0.7) &
+%   scale_y_continuous(breaks = seq(-3.97, -4, -0.01)) &
+%   scale_x_continuous(breaks = seq(-79.06, -79.08, -0.02))
+% ```
+%
+% Now we have seen why spatial resampling methods matter we can take a look at what is implemented in `r mlr3spatiotempcv`.
+% At the time of writing we have implemented 19 different resampling methods for spatial and spatiotemporal analysis.
+% These are categorized into:
+%
+% * Blocking -- Create rectangular blocks in 2D/3D space (as opposed to partitioning in 1D space used in non-spatial methods)
+% * Buffering -- Create buffering zones to remove observations between train and test sets
+% * Spatiotemporal clustering -- Clusters based on coordinates (and/or time-points)
+% * Feature space clustering -- Clusters based on feature space and not necessarily spatiotemporal
+% * Partitioning -- Grouped by factor variables
+%
+% The choice of method may depend on specific characteristics of the dataset and there is no easy rule to pick one method over another, full details of different methods can be found in @Schratz2021 -- the paper deliberately avoids recommending one method over another as the 'optimal' choice is highly dependent on the predictive task, autocorrelation in the data, and the spatial structure of the sampling design.
+% The documentation for each of the implemented methods includes details of each method as well as references to original publications.
+%
+% :::{.callout-tip}
+% ## Spatio*temporal* resampling
+%
+% We have focused on spatial analysis but referred to "spatiotemporal" and "spatiotemp".
+% The resampling methods discussed in this section extend trivially to temporal analysis (or spatial and temporal analysis combined) by setting the `"time"` `col_role` in the task (@sec-performance).
+% See the `r mlr3spatiotempcv` visualization `r link("https://mlr3spatiotempcv.mlr-org.com/articles/spatiotemp-viz.html", "vignette")` for specific details about 3D spatiotemporal visualization.
+% :::
+%
+% ### Spatial Prediction {#sec-spatial-prediction}
+%
+% Until now we have looked at resampling to accommodate spatiotemporal *features*, but what if you want to make spatiotemporal *predictions*?
+% In this case the goal is to make classification or regression predictions for spatial coordinates at a pixel level, usually by creating and labelling raster images.
```

Comment:
```suggestion
% In this case the goal is to make classification or regression predictions at the pixel level, i.e. for an area, defined by the geometric resolution, of a raster image.
```

"Spatial coordinates at pixel level" doesn't go together in this context as a pixel is composed out of multiple coordinates (4) and you would not predict just these and then interpolate. I.e. you don't do "point predictions" but "area predictions" here and one pixel = 1 prediction, 1 coordinate != 1 prediction.